core/Dockerfile.template

```diff
@@ -42,6 +43,8 @@ RUN ln -sf /usr/app/node_modules/balena-cli/bin/balena /usr/bin/balena && \
 COPY contracts contracts
 COPY lib lib
 COPY config config
-COPY entry.sh ./
+# hack to conditionally copy over the ssh key only if it exists
+COPY entry.sh id* ./
+RUN mkdir -p /root/.ssh && cp id /root/.ssh/id_rsa; exit 0
```

Comment:
We should replace this with an env var like `CORE_PRIVATE_KEY` that is set by the Makefile.